### DISCUSSION TITLE
Tail sampling - string_attribute processor based on request header value

### DISCUSSION THREAD

**Discussion by bartez09** [open-telemetry/opentelemetry-collector#8991] [Q&A]
The goal is to save tracing based on a custom requeset header:

1. saving the http request header value via opentelemetry agent:
<pre>
java:
    env:
      - name: OTEL_INSTRUMENTATION_HTTP_SERVER_CAPTURE_REQUEST_HEADERS
        value: forcesampling
</pre>

2. tail sampling:
<pre>
    processors:      
      tail_sampling:
        decision_wait: 10s 
        num_traces: 100
        expected_new_traces_per_sec: 10
        policies:
          [   
            {
              name: test-policy,
              type: string_attribute,
              string_attribute: {key: http.request.header.forcesampling, values: [on]}
            }
          ]
</pre>

Unfortunately, it seems that the key is incorrect (tracing is not saved). Please help me set it up correctly.

**Comment by asbachb** [comment]
@bartez09 could you figure out the correct configuration?
